resources/benchmark-report.mjs

```diff
@@ -16,7 +16,7 @@
     document.head.appendChild(styleElement);
 
     let createTest;
-    const valuesByIteration = new Array();
+    const valuesByIteration = [];
```

Comment:
root format changed this - I can revert if we want to keep the original way.

Reply:
Both are identical to me, so let's keep the root format :-)